test/controllers/ControllerBaseSpec.scala

```diff
@@ -17,7 +17,7 @@
 package controllers
 
 import akka.actor.ActorSystem
-import akka.stream.{ActorMaterializer, Materializer}
+import akka.stream.{ActorMaterializer, Materializer, SystemMaterializer}
```

Comment:
These imports don't seem to actually be used anywhere so could you please remove them?